dia_tap_xero/credentials.py

```diff
@@ -82,6 +82,40 @@ def _write_to_s3(config):
     _upload(_s3_obj(config), json.dumps(creds))
 
 
+def rotate_refresh_tokens(config, json_response):
+    LOGGER.info(config["refresh_token"])
+    LOGGER.info(json_response)
```

Comment:
these fields will contain secrets so we should only log them when developing locally

Reply:
yeah these are old logs. Will remove